boudicca.base/eventcollector-client/src/main/kotlin/base/boudicca/api/eventcollector/debugger/DataShouldContainValue.kt

```diff
@@ -0,0 +1,29 @@
+package base.boudicca.api.eventcollector.debugger
+
+import base.boudicca.api.eventcollector.debugger.color.blue
+import base.boudicca.api.eventcollector.debugger.color.red
+import base.boudicca.api.eventcollector.debugger.color.yellow
+import base.boudicca.model.Event
+
+class DataShouldContainValue(
+    val key: String,
+    val value: Regex,
+    val severity: ValidationSeverity,
+) : EventCollectorValidation {
+    override fun validate(event: Event, verbose: Boolean): ValidationResult {
+        if (!event.data[key]?.contains(value)!!) {
+            when (severity) {
```

Comment:
currently each validator has to replicate this when(). we could move this logic into the ValidationSeverity class with a ValidationSeverity.format("key $key expected to contain $value") call, which then 1) adds the correct Prefix ("WARN: ", "ERR: ", ...) and also can call the correct color function. this would reduce the complexity of each validation